# Goal
string mode

---
# Result
bool success

---
# Feedback
string message
